server/middleware/cached_test.go

```diff
@@ -0,0 +1,67 @@
+package middleware
+
+import (
+	"time"
+
+	"github.com/cloudwan/gohan/schema"
+	"github.com/golang/mock/gomock"
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rackspace/gophercloud"
+)
+
+var _ = ginkgo.Describe("Cached identity service", func() {
```

Comment:
why not the . notation?

Reply:
Because ginkgo declares struct with name "Context", and struct with same name is declared in middleware package. Compiler complains about double declaration.